ipie/utils/chunk_large_chol.py

```diff
@@ -7,7 +7,7 @@ def split_cholesky(ham_filename, nmembers):
     #
     with h5py.File(ham_filename, "r") as source_file:
         dataset = source_file["LXmn"][()]  # pylint: disable=no-member
-        num_chol = dataset.shape[-1]
+        num_chol = dataset.shape[0]
     split_sizes, displacements = make_splits_displacements(num_chol, nmembers)
     assert isinstance(dataset, np.ndarray), "Dataset should be a numpy array"
     dataset = dataset.transpose(1, 2, 0).reshape(-1, num_chol)  # pylint: disable=no-member
```

Comment:
I can't review below here but, please unit test this + remove print statements OR make them more descriptive with a # preceding it so we can grep out everything with #.

Reply:
Why is this pylint ignore here? Is it because it's a HDF5 data set here rather than a numpy array? Is it important to keep it as a dataset rather than copying into memory?

Reply:
I think the pylint is static detector and will report error for the dataset.transpose() operation, if not disabled.

Reply:
Right, my question was more why is it reporting an error. If you read the dataset as (into) a numpy array I imagine  your pylint error would disappear (transpose is a member of a ndarray).